common/src/main/java/org/jboss/hal/testsuite/page/config/TransactionsPage.java

```diff
@@ -1,12 +1,32 @@
 package org.jboss.hal.testsuite.page.config;
 
 import org.jboss.arquillian.graphene.page.Location;
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.finder.FinderNavigation;
 import org.jboss.hal.testsuite.page.ConfigPage;
+import org.jboss.hal.testsuite.page.Navigatable;
+import org.jboss.hal.testsuite.util.ConfigUtils;
 
 /**
  * @author mkrajcov <mkrajcov@redhat.com>
  */
 @Location("#transactions")
-public class TransactionsPage extends ConfigPage {
+public class TransactionsPage extends ConfigPage implements Navigatable {
+
+    public void navigate() {
+        FinderNavigation navigation;
+        if (ConfigUtils.isDomain()) {
+            navigation = new FinderNavigation(browser, DomainConfigEntryPoint.class)
+                    .addAddress(FinderNames.CONFIGURATION, FinderNames.PROFILES)
+                    .addAddress(FinderNames.PROFILE, "full");
```

Comment:
Please use `ConfigUtils.getDefaultProfile()` instead of hardcoded "full" profile name.